build.sh

```diff
@@ -10,9 +10,20 @@ fi
 BUILD_TARGET="$1"
 VERSION=$(cat ./VERSION)
 
+BUILDER="docker build"
+if ! ${BUILDER%% *} -v >/dev/null 2>&1; then
```

Comment:
Using the bash variable substring here just complicates the process without really adding any value - I'd like to keep the script simple to understand.  Just use `if ! docker -v >/dev/null 2>&1; then`